wordpress-mobile/WordPress-Android
# My Site Dashboard: Phase 1 - Quick Start finetune UI

Parent #15189

This PR adjusts the UI based upon the latest design iteration. Included are margin/padding changes, progress circle background color. size ratios, card frame, and font sizes for titles and text.


Dark | Light
-- | --
<img width="250" height="500" alt="Quick Start Dark" src="https://user-images.githubusercontent.com/506707/130638913-5f235d3f-c06a-493c-b67c-64bc3e3a7f54.png"> | <img width="250" height="500" alt="Quick Start Light" src="https://user-images.githubusercontent.com/506707/130638902-d0fc1a88-3db1-488b-b67c-ecffbf03a656.png">


**To test:**
- Launch the app 
- Navigate to My Site -> App Settings -> Test feature configuration
- Enable the `MySiteImprovementsFeatureConfig` and tap the button to restart the app (I also like to swipe the app closed after that and then reopen - sometimes weird behavior happens)
- Log out of the app
- Log in to the app
- When the Quick Start Prompt is shown, tap the "Show me around" option

## Regression Notes
1. Potential unintended areas of impact N/A
2. What I did to test those areas of impact (or what existing automated tests I relied on) N/A
3. What automated tests I added (or what prevented me from doing so) N/A

PR submission checklist:
